### PR TITLE
Makes the augmentation menu not spam the chat

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
this pull request makes it so you dont get spammed with "This body-mod does not match your highest-priority department." a billion times. it now only prints once

## Why It's Good For The Game
this has been a bug for god knows how long. this bug makes the augmentation menu a pain in the ass to use. its gone now

## Changelog
:cl:
fix: Augments menu no longer spams chat
/:cl:

### CODE REVIEW

**Comment by tristain01** [comment]
Oh thank the eyes of discord! About time this got fixed.

File: code/modules/organs/body_modifications.dm [hunk 1]
```diff
@@ -66,7 +66,9 @@ var/global/list/modifications_types = list(
 	if(department_specific.len)
 		if(H && H.mind)
 			var/department = H.mind.assigned_job.department
-			if(!department || !department_specific.Find(department))
+			if(!department)
+				return FALSE
+			else if(!department_specific.Find(department))
```
Comment (Shadow-Quill) [inline code/modules/organs/body_modifications.dm:71]: Don't need the `else` here

**Comment by Alliostra** [comment]
This body-mod does not match your highest-priority department.
This body-mod does not match your highest-priority department.
This body-mod does not match your highest-priority department.

**Comment by ValoTheValo** [comment]
> This body-mod does not match your highest-priority department.
> This body-mod does not match your highest-priority department.
> This body-mod does not match your highest-priority department.

This body-mod does not match your highest-priority department. x30

File: code/modules/organs/body_modifications.dm (hunk 1 above)
Comment (TheShown911) [inline code/modules/organs/body_modifications.dm:71]:
```suggestion
			if(!department_specific.Find(department))
```

Comment (ValoTheValo) [reply code/modules/organs/body_modifications.dm]: so it turns out it doesnt work without the else

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**